src/nbody_simulation_result.jl

```diff
@@ -503,4 +517,51 @@ function msd(sr::SimulationResult{<:WaterSPCFw})
     end
 
     (ts, dr2)
+end
+
+# coordinates should be in angstroms
+function save_to_pdb(sr::SimulationResult{<:WaterSPCFw}, path )
```

Comment:
I think these should be `save` overloads via FileIO.jl style on `.pdb` extensions?